CefGlue/CefRuntime.cs

```diff
@@ -133,6 +113,17 @@ private static void CheckVersion()
 
         private static void CheckVersionByApiHash()
         {
+            // We need to load libCEF.so before getting API Hash on Linux.
+            if (Platform == CefRuntimePlatform.Linux) 
+            {
+                // find all the libcef.so files inside the application folder and its subfolders
+                var found_libs = Directory.GetFiles(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, ".."), libcef.DllName + ".so", SearchOption.AllDirectories);
+
+                // if found, load the first one.
+                if (found_libs.Length > 0)
+                    NativeLibrary.TryLoad(found_libs[0], out _);
```

Comment:
EnumerateFiles should be more efficient
```suggestion
                var libCefFile = Directory.EnumerateFiles(AppDomain.CurrentDomain.BaseDirectory, libcef.DllName + ".so", SearchOption.AllDirectories).FirstOrDefault();

                // if found, load the first one.
                if (libCefFile != null)
                    NativeLibrary.TryLoad(libCefFile, out _);
```